testsuite/cli/src/main.rs

```diff
@@ -109,7 +109,7 @@ fn main() {
                     panic!("Failure to retrieve a waypoint from {}: {}", url_str, e)
                 })
             })
-            .unwrap()
+            .expect("Must provide either '--waypoint' or '--waypoint-url'")
```

Comment:
Use this instead: https://github.com/libra/libra/blob/master/config/management/src/config.rs#L21